1_sanskr/tei/sa_kaivalyopaniSad.xml

```diff
@@ -312,153 +312,149 @@
 
 <text xml:lang="sa-Latn">
   <body>
-<div>
-<p>[Oṃ bhadraṃ karṇebhiḥ śṛṇuyāma devāḥ bhadraṃ paśyemākṣabhir yajatrāḥ |
+
+<p xml:id="Kai.00.a">[Oṃ bhadraṃ karṇebhiḥ śṛṇuyāma devāḥ bhadraṃ paśyemākṣabhir yajatrāḥ |
```

Comment:
I'm not sure if this was the right way to treat the shanti mantras (as well as the extra / variant lines later, see other tags labeled "a" / "b" / "c"), what do you think @akprasad ?